test/functional/test_framework/blocktools.py

```diff
@@ -196,6 +198,20 @@ def create_raw_transaction(node, txid, to_address, *, amount):
     assert_equal(signresult["complete"], True)
     return signresult['hex']
 
+def create_tx_with_large_script(prevtx, n, scriptPubKey, amt1=1, amt2=0.1 ):
+    tx = CTransaction()
+    tx.vin.append(CTxIn(COutPoint(prevtx, n), b"", 0xffffffff))
+    tx.vout.append(CTxOut(int(amt1 * COIN), scriptPubKey))
+    #tx.vout.append(CTxOut(int(amt1 * COIN), scriptPubKey))
```

Comment:
Please remove the commented out part if unnecessary.

Reply:
This is leftover code from trying to increase the number of spendable outputs and reach the prune limit sooner. But it was not sustainable, most outputs became dust and it increased the overall time.